contents/handbook/company/culture.md

```diff
@@ -6,30 +6,28 @@ showTitle: true
 
 So, what's it like working at PostHog?
 
-<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/rRwzJiljpSA" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+## We're 100% remote 
 
-## All remote
+[Our team](/people) is 100% remote, and distributed across more than 15 countries.
```

Comment:
@smallbrownbike we should automate this number

Reply:
There's some hardcoded reference in [contents/handbook/company/grown-ups.md](https://github.com/posthog/posthog.com/tree/master/contents/handbook/company/grown-ups.md) as well